source/imports/domain/banking-domain-module.js

```diff
@@ -0,0 +1,15 @@
+import BankAccountRouter from '../es-routers/bank-account-router';
+
+const BankingDomain = Space.Module.define('BankingDomain', {
+
+  requiredModules: [
+    'Space.eventSourcing'
+  ],
+
+  singletons: [
+    BankAccountRouter
```

Comment:
Yeah a module might be overkill for this project … and i guess we are looking for ways to avoid the setup ceremony in the long run.